internal/pkg/describe/rd_web_service.go

```diff
@@ -86,6 +86,7 @@ func (d *RDWebServiceDescriber) Describe() (HumanJSONStringer, error) {
 			Port:        service.Port,
 			CPU:         service.CPU,
 			Memory:      service.Memory,
+			Platform:    "LINUX/X86_64",
```

Comment:
An alternative to defaulting a platform string would be to not show the column at all if we do not have the information 🤔  

For RD web service, I feel like user wouldn't be interested in the platform information anyway since they should have known that the only valid platform is "linux/x86_64" if they have tried to use any other platform for the service.

Reply:
I have removed the `Platform` column for RDWS because you're right-- it's the only option, so it wouldn't be useful.